aptos-move/aptos-vm/src/move_vm_ext/warm_vm_cache.rs

```diff
@@ -101,6 +101,22 @@ struct WarmVmId {
     natives: Bytes,
     vm_config: Bytes,
     core_packages_registry: Option<Bytes>,
+    optional_id: Option<OptId>,
```

Comment:
This should work according to previous discussion, although I think it's more straightforward to do just
``` Rust
bin_v7_enabled: bool
```

Reply:
@runtian-zhou and I discussed about it and think directly adding a bool field would be too hacky for production code. What do you think?

Reply:
I think the other way around, 
, plus: this is in-mem only, we can safely change the implementation any time
, plus: `Option<OptId>` takes more memory than `bool`